models/tmp/stg_stripe__pricing_tmp.sql

```diff
@@ -0,0 +1,40 @@
+{{ config(enabled=var('stripe__using_subscriptions', True)) }}
+
+select * from (
+
+{% if var('stripe__using_price', does_table_exist('price')) %}
+
+{{
+    fivetran_utils.union_data(
+        table_identifier='price', 
+        database_variable='stripe_database', 
+        schema_variable='stripe_schema', 
+        default_database=target.database,
+        default_schema='stripe',
+        default_variable='price',
+        union_schema_variable='stripe_union_schemas',
+        union_database_variable='stripe_union_databases'
+    )
+
+}}
+
+{% else %}
+
+{{
+    fivetran_utils.union_data(
+        table_identifier='plan', 
+        database_variable='stripe_database', 
+        schema_variable='stripe_schema', 
+        default_database=target.database,
+        default_schema='stripe',
+        default_variable='plan',
+        union_schema_variable='stripe_union_schemas',
+        union_database_variable='stripe_union_databases'
+    )
+
+}}
+
+{% endif %}
+
+) as fields
+{{ livemode_predicate() }}
```

Comment:
Let's make the same update you made in the subscription tmp model where you consolidated this logic into one block in order to make the code it bit more dry.

Reply:
done!